### PR TITLE
Rough in partials. Change naming convention.

### PR DESCRIPTION
**Pull request by DouglasUrner**

### CODE REVIEW

File: app/views/home_teacher/_assessment_tab.html.erb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<%
+ # app/views/home_teacher/_assessment_tab.html.erb
+ %>
+<div class='home-teacher'>
+<% @periods.each do |p| %>
+
+  <div class='class-title'>
+    <a href=#<%= "S#{p[:session]}P#{p[:period]}" %> data-toggle='collapse'>
```
Comment (Gaelan) [inline app/views/home_teacher/_assessment_tab.html.erb]: This "class-period" thing appears quite a bit. Probably worth drying up.

File: app/views/home_teacher/_overview_tab.html.erb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <%
- # app/views/home_teacher/index.html.erb
+ # app/views/home_teacher/_overview.html.erb
```
Comment (Gaelan) [inline app/views/home_teacher/_overview_tab.html.erb]: You're one step behind…

File: app/views/home_teacher/_assessment_tab.html.erb (hunk 1 above)
Comment (DouglasUrner) [inline app/views/home_teacher/_assessment_tab.html.erb]: Always. There is a log of sogginess here still.

File: app/views/home_teacher/_overview_tab.html.erb (hunk 2 above)
Comment (DouglasUrner) [inline app/views/home_teacher/_overview_tab.html.erb]: Not sure I understand that…

Comment (Gaelan) [inline app/views/home_teacher/_overview_tab.html.erb]: You fixed the comment… then renamed the file.

**Pull request merged**